drivers/charger/charger_bq25180.c

```diff
@@ -262,6 +331,12 @@ static int bq25180_init(const struct device *dev)
 		return ret;
 	}
 
+	ret = bq25180_set_charge_voltage(dev, cfg->initial_voltage_microvolt);
+	if (ret < 0) {
+		LOG_ERR("Could not set the target voltage.");
```

Comment:
I think it's worth logging the return code.

Reply:
Done